kano_updater/status.py

```diff
@@ -190,6 +194,19 @@ def last_check_urgent(self, value):
 
         self._last_check_urgent = value
 
+    # -- first_boot_countdown - used to stop updates for a set amount of time
+    @property
+    def first_boot_countdown(self):
+        return self._first_boot_countdown
+
+    @first_boot_countdown.setter
+    def first_boot_countdown(self, value):
+        if not isinstance(value, int):
```

Comment:
A slightly more _pythonic_ (yeah, it's a horrible word) to do this would be:

```
try:
   self._first_boot_countdown = int(value)
except ValueError:
  ...
```